Makefile

```diff
@@ -63,7 +63,7 @@ BAZEL_OPTS +=	--sandbox_writable_path=$${XDG_RUNTIME_DIR}
 $(shell [ -d $(XDG_RUNTIME_DIR) ] || mkdir -p $(XDG_RUNTIME_DIR))
 endif
 
-ci: all
+ci: all tidy vendor bazel-generate generate-verify
```

Comment:
the `vendor` part can be problematic in some cases but as long as we do it in the `ci` target that's ok, if we encounter an issue - we can drop it later